app/assets/javascripts/govuk_publishing_components/analytics-ga4/gtm-page-views.js

```diff
@@ -48,15 +50,15 @@
     },
 
     getLocation: function () {
-      return document.location.href
+      return this.PIIRemover.stripPII(document.location.href)
     },
 
     getReferrer: function () {
-      return document.referrer
+      return this.PIIRemover.stripPIIWithOverride(document.referrer, true, true)
```

Comment:
@andysellick This is a consequence of what the implementation guidance said for this value - since referrer talked about worst case scenario redaction, Tara and I agreed to use date and postcode stripping on the referrer.

Reply:
Okay, that's fine. I still think this is inconsistent but I don't think it should hold up this PR. I'll make a note and we can come back to it later.